lib/tasks/after_tasks.rake

```diff
@@ -223,44 +223,6 @@ namespace :After do
     end
   end
 
-  desc "Convert remaining chapter kudos into work kudos"
```

Comment:
The tests for this broke, and rather than investigate I thought removing would be fine since I recently ran this

Reply:
Rails 7.0 removes `errors.keys`, it is replaced by `errors.attribute_names`.  If that's the only problem with this test then fixing it should be rather simple.

(Currently the test warns about this: `DEPRECATION WARNING: ActiveModel::Errors#keys is deprecated and will be removed in Rails 7.0.`)

Reply:
That does fix the issue, thanks! However, I still think we should remove this to avoid confusion, since it's really a 1-off task that won't need to be run again now

Reply:
Usually I think it would be better to keep it, for the following usecase: Somebody else who is using the archive software might still need to run the task, and there would be no technical reason to force them to do so before upgrading to Rails 7.

But chapter kudos haven't been a thing since before 2011, so that does not apply here.